api.swagger.yaml

```diff
@@ -1845,6 +1845,32 @@ paths:
           description: The UUID or request body JSON was malformed.
         '404':
           description: No manual page with that UUID was found.
+  '/manuals/{manual_page_id}/pdf-export':
+    get:
```

Comment:
GET requests don't have a request body. Changing it to a POST would be acceptable, or if you want to keep it as a GET for semantic correctness, we should move the secret token to a separate header (`Authorization`, perhaps?) as I don't want it to be a path or query parameter.

Reply:
I wasn't aware that GET requests **couldn't** have bodies. I'd rather stick with GET and move the token to a header, but have we established a header we can use?

Reply:
Yes, we seem to have settled on:

```
Authorization: Bearer secret-token-value
```

Reply:
Done.